membership_policy.md

```diff
@@ -128,6 +130,82 @@ The following apply to the part of project for which one would be an approver in
   * Mentor contributors and reviewers
   * May approve contributions for acceptance
 
+## Special Interest Group (SIG) Chair
+
+The KubeVirt project is organized primarily into SIGs, each with a common purpose of advancing the project with respect to a specific topic, such as Networking or Storage. A SIG Chair is required to help guide and coordinate the SIG. 
+
+### Requirements
+
+* Active reviewer for SIG matters for at least 3 months
+* Track record of contributions to SIG matters
+* Advanced knowledge in SIG subject
+
+### Responsibilities and privileges
```

Comment:
What about its responsibility over the SIG subprojects?

Reply:
So you are asking about making the hierarchy mentioned explicitly? I thought that was already clear?

Reply:
A SIG TL has the responsibility to create subprojects [1].
As we do not define a TL, I guess it is embedded into the chair role.

> MUST Approve & facilitate the creation of new subprojects

[1]  https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#tech-lead

Reply:
My intent here was that the subproject lead is responsible for creating and maintaining the subproject.

Reply:
^^ @EdDev

Reply:
@dhiller , I would just add the quoted point to the list.

> Approve & facilitate the creation of new subprojects

Reply:
Yeah, really good point - I'll update the PR. Thanks!